Fliglio/Chinchilla/RPCPublisher.php

```diff
@@ -37,6 +54,10 @@ public function publishReply(Message $msg, MappableApi $api) {
 
 		$worker = new WorkerPublisher($this->connection, $queueName);
 
+		foreach ($filters as $filter) {
+			$worker->addFilter($filter);
```

Comment:
should we not just add `$filters` as a constructor argument for `WorkerPublisher`?